mpp/mpp_io.F90

```diff
@@ -384,7 +384,7 @@ module mpp_io_mod
   public :: mpp_get_dimension_length, mpp_get_axis_bounds
 
   !--- public interface from mpp_io_misc.h ----------------------
-  public :: mpp_io_init, mpp_io_exit, netcdf_err, mpp_flush, mpp_get_maxunits, do_cf_compliance
```

Comment:
can you add this back? still needs to be there so its public for other components